pkgs/modules/interpreters/nodejs/default.nix

```diff
@@ -36,6 +36,14 @@ with pkgs.lib; {
   };
 
   config = mkIf cfg.enable {
+    checks.nodejs = pkgs.writeShellScript "nodejs-check" ''
+      output=$(${nodejs-wrapped}/bin/node --version)
+      if [ "$output" != "v${nodejs-wrapped.version}" ]; then
+        echo "Node.js version mismatch: expected v${nodejs-wrapped.version}, got $output"
+        exit 1
+      fi
```

Comment:
this test could've caught a bug in a previous release of v1!

Reply:
Yes, this will be very important when we enforce version pinning. Do not want to accidentally upgrade the version by upgrading a Nix channel.